pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -948,6 +894,86 @@ func (az *Cloud) reconcileLoadBalancer(clusterName string, service *v1.Service,
 	return lb, nil
 }
 
+func (az *Cloud) createLoadBalancerRule(
+	expectedProbes *[]network.Probe,
+	expectedRules *[]network.LoadBalancingRule,
+	port v1.ServicePort,
+	service *v1.Service,
+	lbFrontendIPConfigID string,
+	lbBackendPoolID string,
+	lbName string,
+	lbIdleTimeout *int32) error {
```

Comment:
nit: move `expectedProbes` and `expectedRules` to return so that it's much more easy to read

Reply:
I have moved all lb rule and probe creation logic into a new func, that would be more clear